### PR TITLE
Styles cat's card 

### PR DESCRIPTION
**Pull request by MissMelisa**
add some styles to the cat's card in the main presentation
image

### CODE REVIEW

File: src/Pages/CatPresentation/index.jsx [hunk 1]
```diff
@@ -61,7 +62,34 @@ function CatPresentationPage() {
...
+    isFetching,
+    fetchNextPage,
+    isFetchingNextPage,
+  } = useInfiniteQuery("breeds", fetchProjects, {
+    getNextPageParam: () => {
+      return page;
+    },
+    getPreviousPageParam: () => {
+      return page - 1;
+    },
```
Comment (falecci) [inline src/Pages/CatPresentation/index.jsx:79]: we don't need this

File: src/Pages/CatPresentation/index.jsx [hunk 2]
```diff
@@ -61,7 +62,34 @@ function CatPresentationPage() {
...
+  });
+
+  function handleOnNextClick() {
+    setPage(page + 1);
+  }
```
Comment (falecci) [inline src/Pages/CatPresentation/index.jsx:84]: We can think of a better name for this, since we are not _clicking_.

**Review by falecci — CHANGES_REQUESTED** [review]

File: src/Pages/CatPresentation/index.jsx (hunk 1 above)
Comment (MissMelisa) [reply src/Pages/CatPresentation/index.jsx]: checked it

File: src/Pages/CatPresentation/index.jsx (hunk 2 above)
Comment (MissMelisa) [reply src/Pages/CatPresentation/index.jsx]: checked it

**Review by falecci — APPROVED** [review]

**Pull request merged**